src/components/patch/PatchContainer.tsx

```diff
@@ -230,6 +230,11 @@ export class PatchContainer extends React.Component<Props, State> {
           username = urlParams.get("user");
         }
       }
+      const pathname = window.location.hash.split("?")[0].split('/')
+      let projectName = ''
+      if(pathname.length === 4 && pathname[2] === 'project') {
```

Comment:
We didn't discuss how we wanted to take in parameters, but I think if the current URL is something like `https://spruce.mongodb.com/#/patches?user=john.liu` for a user's patches, `https://spruce.mongodb.com/#/patches?project=spruce` should be for a project's patches. It seems like you're trying to accommodate both query params here, but I think that's actually invalid input

Reply:
I adjusted the routes `patches/project/{projectName}` and `patches/user/{username}`. If the username is empty, the page will fetch the currently logged in user's patches.